viite-UI/src/model/ProjectCollection.js

```diff
@@ -324,25 +344,25 @@
       }).last().value();
       var isNewRoad = changedLink.status == LinkStatus.New.value;
 
-      if(isNewRoad && !validUserGivenAddrMValues(_.first(dataJson.linkIds), dataJson.userDefinedEndAddressM)){
+      if(isNewRoad && !validUserGivenAddrMValues(_.first(dataJson.ids || dataJson.linkIds), dataJson.userDefinedEndAddressM)){
         new GenericConfirmPopup("Antamasi pituus eroaa yli 5% prosenttia geometrian pituudesta, haluatko varmasti tallentaa tämän pituuden?", {
           successCallback: function () {
-            createOrUpdate(dataJson, changedLinks);
+            createOrUpdate(dataJson);
           },
           closeCallback: function () {
             applicationModel.removeSpinner();
             eventbus.trigger('roadAddress:projectLinksUpdated');
           }
         });
       } else{
-        createOrUpdate(dataJson, changedLinks);
+        createOrUpdate(dataJson);
       }
       return true;
     };
 
     this.preSplitProjectLinks = function(suravage, nearestPoint){
       applicationModel.addSpinner();
-      var form = $('#roadAddressProjectFormCut');
+      //var form = $('#roadAddressProjectFormCut');
```

Comment:
If the #roadAddressProjectFormCut is not needed then is it possible to be removed?